examples/list-box-sample/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "fast-todo-app",
```

Comment:
Probably want to change this name?

Reply:
Ooops. Copy/paste error. Will fix it up.